src/Commands/Serve/index.js

```diff
@@ -164,6 +165,19 @@ class Serve extends Command {
     }
 
     const nodemon = require('nodemon')
+    const foldersToIgnore = [
```

Comment:
I believe, we can make it little bit more organized. Also their is no need to add `*` to user defined ignore values, since it removes the flexibility of defining custom blobs.

The following feels more approachable to me

```js
['/tmp/*', '/resources/*', '/public/*'].concat(ignore || []).map((folder) => `${process.cwd()}/${folder}`)
```